### PR TITLE
UCT/ROCM: fix test_md_fork

### PR DESCRIPTION
**Pull request by edgargabriel**
## What
fix for the test_md_fork test of gtest for ROCm devices and memory domains.

### CODE REVIEW

File: test/gtest/uct/test_md.cc [hunk 1]
```diff
@@ -1067,7 +1067,11 @@ UCS_TEST_SKIP_COND_P(test_md_fork, fork,
         memset(buf, 42, REG_SIZE);
         /* child touch the page */
         EXPECT_EQ(0, memcmp(page, buf, REG_SIZE));
+#if HAVE_ROCM
+        _exit(0);
+#else
         exit(0);
+#endif
```
Comment (yosefe) [inline test/gtest/uct/test_md.cc:1074]: why needed?

Comment (yosefe) [reply test/gtest/uct/test_md.cc:1074]: @edgargabriel can we avoid this in the code and solve in a better way?

Comment (edgargabriel) [reply test/gtest/uct/test_md.cc:1074]: @yosefe yes, I was told that the runtime fix have a solution for the fork cleanup problem implemented, and I am just waiting to be able to test that. If that works indeed, I would just keep the first part of the PR and remove the '_exit' from it

Comment (yosefe) [reply test/gtest/uct/test_md.cc:1074]: > @yosefe yes, I was told that the runtime folks have a fix for the fork cleanup problem implemented, and I am just waiting to be able to test that. If that works indeed, I would just keep the first part of the PR and remove the '_exit' from it

sounds good

**Comment by edgargabriel** [comment]
@yosefe I've updated the pr to contain only the change to the rcache_params, and removed the change to gtest. Please have look when you have a chance.

**Review by yosefe — APPROVED** [review]
pls squash

**Pull request merged**